tests/session/session-debug-006.phpt

```diff
@@ -0,0 +1,74 @@
+--TEST--
+MongoDB\Driver\Session debug output (with transaction options)
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_crypto(); ?>
+<?php skip_if_not_live(); ?>
+<?php skip_if_server_version('<', '3.6'); ?>
```

Comment:
This SKIPIF was copied from a previous test file, but it's not sufficient for this case. Since you're calling `startTransaction` here, you should look to the SKIPIF logic in that method's corresponding tests:

```
<?php skip_if_not_libmongoc_crypto() ?>
<?php skip_if_no_transactions(); ?>
```

Note: this differs from the SKIPIF logic in `session-debug-005.phpt`, which uses:

```
<?php skip_if_not_mongos_with_replica_set(); ?>
<?php skip_if_server_version('<', '4.1.6'); ?>
<?php skip_if_not_clean(); ?>
```

That file is specifically testing transactions on a sharded cluster. The combination of `skip_if_not_mongos_with_replica_set` and `skip_if_server_version('<', '4.1.6');` is actually more restrictive than `skip_if_no_transactions`, which you can verify by reading through `skipif.php` and `tools.php`. That said, `session-debug-005.phpt` _does_ appear to be missing `skip_if_not_libmongoc_crypto();` so if you could add that to its SKIPIF that'd be great. In practice, we don't test libmongoc w/o crypto (needed to generate session IDs), but it's reasonable to have that checked for consistency (and self-documentation).